src/3rdparty/walkontable/test/spec/settings/stretchH.spec.js

```diff
@@ -69,12 +69,15 @@ describe('stretchH option', () => {
     const evt = document.createEvent('CustomEvent'); // MUST be 'CustomEvent'
     evt.initCustomEvent('resize', false, false, null);
     window.dispatchEvent(evt);
+    wt.draw();
 
     await sleep(300);
 
     const currentTableWidth = $table.outerWidth();
     expect(currentTableWidth).toBeAroundValue($table[0].clientWidth);
     expect(currentTableWidth).toBeGreaterThan(initialTableWidth);
+
+    done();
```

Comment:
`done()` call is not necessary when `async/await` is used.

Reply:
Without `done` the test would timeout before the async part was finished:
![image](https://user-images.githubusercontent.com/7397134/60669365-73543380-9e6e-11e9-84e7-287c90ecd8bf.png)

After updating Jasmine to the latest version this seem to not happen anymore, so I removed it.